Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_smorci/src/main.rs

```
@@ -0,0 +1,70 @@
+use num_bigint::BigUint;
+use std::io;
+use std::str::FromStr;
+
+const NUMBER_OF_DAYS_IN_A_YEAR: u32 = 365;
+
+fn convert_years_to_days(years: BigUint) -> BigUint {
+    NUMBER_OF_DAYS_IN_A_YEAR * years
+}
+
+fn convert_string_to_biguint(string: &mut String) -> BigUint {
+    let result = BigUint::from_str(string.trim()).expect("Error parsing the input to unsigned integer. Maybe the number you entered is negative, or it has alphabetical characters");
+    result
+}
+
+fn get_input(buffer: &mut String) -> io::Result<()> {
+    let stdin = io::stdin();
+
+    stdin.read_line(buffer)?;
```

i think this silently ignores error.

---

i looked it up, `?` returns with Err immediately, so this doesn't ignore the error